cerebrium/cortex/advanced-functionality/persistent-storage.mdx

```diff
@@ -31,3 +31,25 @@ model = torch.jit.load(file_path)
 ```
 
 Now, in subsequent deployments, the model will load from the cache rather than download it again.
+
+## Increasing your Persistent Storage Size
+
+<Note>Once increased, your persistent storage size cannot be decreased.</Note>
+
+By default, your account is given 50GB of persistent storage to start with. However, if you find you need more (for example, you get an error saying `disk quote exceeded`) then you can increase your allocation using the following steps:
```

Comment:
50GB or 20GB?

Reply:
It's 50GB at the moment. 
20GB won't fit a 7B model.